src/test/resources/network/brightspots/rcv/test_data/stop_tabulation_early_test/stop_tabulation_early_test_config.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
To make this test, I copied and pasted another directory with slight modifications. Is that the recommended way to go about this? If I were being a little more strict, I'd test, e.g.:
1. Invalid inputs (e.g. "-1" in the config)
2. Config set to zero (stops tabulating successfully before any tabulation, i.e. just the config is spit out)
3. Config set to > than the number of rounds provided

But with a new directory and large files needed for each test, this feels like overkill. I believe I've followed the existing best-practices laid out in the repo, but let me know if anybody has other preferences here.

Reply:
The other tests won't flip out, but it will force you to re-save the config when loading the test, before tabulating. I didn't actually realize that's what was sometimes forcing me to re-save until I saw this comment. It is annoying, but not terrible? I'm happy to manually update all configs if needed, though perhaps we should wait til the end of 1.4 work so we only need to update each config once.

Reply:
Ok, that sounds reasonable to me, filed #642 to remember this.